SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -142,23 +142,24 @@ Interest InterestFactory::CreatePartitionInterest(const UAbstractLBStrategy* LBS
 
 	// Ensure server worker receives AlwaysRelevant entities.
 	PartitionQuery = Query();
-	PartitionQuery.ResultComponentIds = ServerNonAuthInterestResultType;
+	PartitionQuery.ResultComponentIds = ServerNonAuthInterestResultType.ComponentIds;
+	PartitionQuery.ResultComponentSetIds = ServerNonAuthInterestResultType.ComponentSetsIds;
 	PartitionQuery.Constraint = CreateServerAlwaysRelevantConstraint();
-	AddComponentQueryPairToInterestComponent(PartitionInterest, SpatialConstants::WELL_KNOWN_COMPONENT_SET_ID, PartitionQuery);
+	AddComponentQueryPairToInterestComponent(PartitionInterest, SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID, PartitionQuery);
```

Comment:
is partition entity an actor, or should this use the known entity component set?

Reply:
Yep good point - will switch